pkg/net/http/httputils.go

```diff
@@ -27,6 +28,7 @@ type QueryHeader struct {
 	SortOrder    string
 	StartDate    time.Time
 	EndDate      time.Time
+	Alias        string
```

Comment:
Validate if alias should be +1

```suggestion
	Aliases        []string
```